app/controllers/UpfrontPaymentController.scala

```diff
@@ -88,11 +88,12 @@ class UpfrontPaymentController @Inject() (
       case j: Journey.BeforeAnsweredCanPayUpfront =>
         logErrorAndRouteToDefaultPage(j)
 
-      case j: Journey.AfterAnsweredCanPayUpfront if !j.canPayUpfront.userCanPayUpfront =>
-        logErrorAndRouteToDefaultPage(j)
-
-      case j: Journey.AfterAnsweredCanPayUpfront if j.canPayUpfront.userCanPayUpfront =>
-        finalStateCheck(j, displayUpfrontPaymentAmountPage(request.eligibilityCheckResult, Left(j)))
+      case j: Journey.AfterAnsweredCanPayUpfront =>
```

Comment:
compiler made me do this, I prefer it as it was before, but it thinks there's possible a missed case, which I guess there technically could be but 🤷